management/src/test/java/org/ehcache/management/providers/statistics/StatsUtil.java

```diff
@@ -1,71 +0,0 @@
-/*
```

Comment:
oh that's weird, you moved it on purpose from test to main
can you please explain why ?